app/src/main/java/org/eyeseetea/malariacare/domain/entity/Settings.java

```diff
@@ -4,20 +4,16 @@
 
 public class Settings {
 
-    public enum MediaListMode  { GRID, LIST }
-    public final MediaListMode DEFAULT_LIST_STYLE = MediaListMode.GRID;
+    private final MediaListMode DEFAULT_LIST_STYLE = MediaListMode.GRID;
 
     private String systemLanguage;
     private String currentLanguage;
     private MediaListMode mediaListMode;
 
-    public Settings(String systemLanguage, String currentLanguage) {
+    public Settings(String systemLanguage, String currentLanguage, MediaListMode mediaListMode) {
         this.systemLanguage = required(systemLanguage, "systemLanguage is required");
         this.currentLanguage = currentLanguage;
-    }
-
-    public Settings(MediaListMode mediaListMode) {
```

Comment:
A constructor with only mediaListMode has no sense because systemLanguage and currentLanguage are mandatories and we should not be able to create Settings in an inconsistent state